Makefile

```diff
@@ -702,13 +702,16 @@ lint-go:
 	golangci-lint run -c .golangci.yml --build-tags='$(LIBFIDO2_TEST_TAG) $(TOUCHID_TAG) $(PIV_TEST_TAG)' $(GO_LINT_FLAGS)
 
 .PHONY: fix-imports
-fix-imports: GOLANG_LINT_ARGS = --build-tags='$(LIBFIDO2_TEST_TAG) $(TOUCHID_TAG) $(PIV_TEST_TAG)' --fix
 fix-imports:
-	golangci-lint run -c .golangci.yml $(GOLANG_LINT_ARGS)
-	cd api/ && golangci-lint run -c ../.golangci.yml $(GOLANG_LINT_ARGS)
-	cd build.assets/tooling && golangci-lint run -c ../../.golangci.yml $(GOLANG_LINT_ARGS)
-	cd .cloudbuild/scripts && golangci-lint run -c ../../.golangci.yml $(GOLANG_LINT_ARGS)
-	cd assets/backport && golangci-lint run -c ../../.golangci.yml $(GOLANG_LINT_ARGS)
+	make -C build.assets/ fix-imports
+
+.PHONY: fix-imports/host
+fix-imports/host:
+	@if ! type gci 2>&1 >/dev/null; then\
+		echo "gci is not installed or it's missing from PATH, consider installing it 'go install github.com/daixiang0/gci@latest' or use 'make -C build.assets/ fix-imports'";\
+		exit 1;\
+	fi
+	gci write -s 'standard,default,prefix(github.com/gravitational/teleport)' --skip-generated .
```

Comment:
Nice, I didn't realize `gci write .` worked.

Reply:
`**/*.go` doesn't work in `Makefile` as this pattern goes only one level deep (the same as `*/*.go`) It's super confusing as `gci` runs without an error but doesn't do much.